back-end/app/Http/Controllers/Follow/FollowsController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers\Follow;
+
+use App\Http\Controllers\Controller;
+use App\Models\Follow;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class FollowsController extends Controller
+{
+    // Get all follows activity in follows table
+    public function index()
+    {
+        $follows = Follow::all();
+
+        return response()->json(['data' => $follows], 200);
+    }
+
+    // Get all user's followings
+    public function followings($follower_id)
+    {
+        $followings = Follow::where('user_id', $follower_id)->get();
+
+        return response($followings);
+        return response()->json(['data' => $followings], 200);
+    }
+
+    // Get all user's followers
+    public function followers($following_id)
+    {
+        $followers = Follow::where('following_id', $following_id)->get();
+
+        return response($followers);
+        return response()->json(['data' => $followers], 200);
+    }
+
+    // Follow user
+    public function store(Request $request)
+    {
+        $follower = $request['follower_id'];
+        $following = $request['following_id'];
+
+        $user = User::find($following);
+
+        if (!$user) {
+            return response()->json(['message' => 'User does not exist'], 404);
+        }
+
+        if ($follower === $following) {
+            return response()->json(['message' => 'We don\'t do that here'], 200);
+        }
+
+        $following_list = Follow::where('user_id', $follower)
+            ->where('following_id', $following)
```

Comment:
```suggestion
        $following_list = Follow::where([
             'user_id' => $follower,
             'following_id' => $following
        ])
```
Alternative way if you have many where clause

Reply:
Nice! will do this as wel.